lib/stat_tracker.rb

```diff
@@ -1 +1,131 @@
+require "csv"
 
+class StatTracker
+  attr_accessor :games_data, :teams_data, :game_teams_data
+
+  def self.from_csv(locations)
+    games_data = CSV.read(locations[:games], headers: true, header_converters: :symbol)
+    teams_data = CSV.read(locations[:teams], headers: true, header_converters: :symbol)
+    game_teams_data = CSV.read(locations[:game_teams], headers: true, header_converters: :symbol)
+
+    new(games_data, teams_data, game_teams_data)
+  end
+
+  def initialize(games_data, teams_data, game_teams_data)
+    @games_data = games_data
+    @teams_data = teams_data
+    @game_teams_data = game_teams_data
+  end
+
+  def highest_total_score
```

Comment:
looks like similar code in both #highest_total_score and #lowest_total_score; opportunity to refactor in iteration 4. 
Perhaps get all total scores into array and then call #max_by and #min_by to find highest and lowest

Reply:
closed with helper